.github/workflows/_create-alpha-release-tag.yml

```diff
@@ -0,0 +1,37 @@
+name: Create alpha_release tag
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  tag-release:
+    needs: unit-tests
+    runs-on: ubuntu-20.04
+
+    permissions:
+      contents: write
```

Comment:
Given this has repo write permissions, we need to be a little careful when we invoke this from elsewhere. [Github recommends pinning to a ref](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow) - `@master` is probably fine - changes to this file will have been code-reviewed etc. I don't think we can enforce this here, but perhaps we could add a comment with an example of expected usage? Something like

```
# Usage: 
# jobs:
#  call-create-alpha-release-from-pay-ci:
#    uses: alphagov/pay-ci/.github/workflows/_create-alpha-release-tag.yml@master
```

Reply:
Sure, adding some help text is fine, although as I understand it pinning to @master is the same as not pinning, except in the case that the primary branch name is changed.

Reply:
Just an aside again, about how much I dislike the github permissions model :( I really wish it were more granular so you could grant "make a release" type permissions without requiring access to the whole repo :(

Reply:
Added to the top of the workflow (I've actually added help info to both since it's a good idea! :) )